.github/workflows/pr.yaml

```diff
@@ -304,3 +305,14 @@ jobs:
         node_type: cpu4
         build_type: pull-request
         run_script: "ci/cudf_pandas_scripts/pandas-tests/diff.sh"
+  third-party-integration-tests-cudf-pandas:
```

Comment:
We only want to run the integration tests nightly (not on a per PR basis), so I'll remove when this PR is approved.